src/vxfw/Border.zig

```diff
@@ -5,10 +5,20 @@ const Allocator = std.mem.Allocator;
 
 const vxfw = @import("vxfw.zig");
 
+pub const BorderLabel = struct { text: []const u8, alignment: enum {
+    TopLeft,
+    TopCenter,
+    TopRight,
+    BottomLeft,
+    BottomCenter,
+    BottomRight,
+} };
```

Comment:
Let's change these enums to snake case (eg. `top_left`) to match zig coding style.